README.rst

```diff
@@ -25,9 +25,17 @@ Kinto
 .. |pypi| image:: https://img.shields.io/pypi/v/kinto.svg
     :target: https://pypi.python.org/pypi/kinto
 
+What JSON is?
+================
+
+JSON is a popular and easy to use markup language.
+
+What Kinto is capable of?
+=========================
```

Comment:
Why not add this at the end of the introduction? 

I don't think `What JSON is ?` should be the introductory phrase